src/index.scss

```diff
@@ -0,0 +1,235 @@
+@mixin center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+}
+
+body {
+  margin: 0;
+  color: #ffeead;
+  background: #ffeead;
+}
+
+.containter {
+  display: grid;
+  grid-template-areas:
+    'logo     header'
+    'sidebar  content'
+    'sidebar  content'
+    'footer   footer';
+  grid-template-rows: 50px minmax(calc(50vh - 75px), 100%) minmax(calc(50vh - 75px), 100%) 50px;
+  grid-template-columns: 150px 1fr;
+  grid-gap: 5px;
+  min-height: calc(100vh - 30px);
+  margin: 15px;
+  font-family: monospace;
+  font-size: 22px;
+  font-weight: 600;
+
+  &__logo {
+    grid-area: logo;
+    background: #96ceb4;
+
+    &-title {
+      @include center;
+
+      text-transform: uppercase;
+    }
+  }
+
+  &__header {
+    grid-area: header;
+    background: #ff706a;
+  }
+
+  &__content {
+    grid-area: content;
+    background: #fecc5d;
+  }
+
+  &__sidebar {
+    grid-area: sidebar;
+    background-color: #00cc9a;
+  }
+
+  &__footer {
+    grid-area: footer;
+    background-color: #df9edf;
+
+    &-title {
+      @include center;
+
+      margin: 0;
+      text-transform: uppercase;
+    }
+  }
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 10px;
+  text-transform: uppercase;
+
+  &__list {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    padding: 10px 0;
+    list-style: none;
+  }
+
+  &__link {
+    padding: 0 10px;
```

Comment:
More semantic way will be to add margin-left to all header__item elements, except the first one. In this case you will avoid extra spacing on the first element, and to add space between elements better use 'margin'. 'padding' is used for spacing inside an element.

Reply:
ok